server/routerlicious/packages/routerlicious-base/src/riddler/runnerFactory.ts

```diff
@@ -47,6 +50,25 @@ export class RiddlerResources implements IResources {
 
 export class RiddlerResourcesFactory implements IResourcesFactory<RiddlerResources> {
     public async create(config: Provider): Promise<RiddlerResources> {
+        // Cache connection
+        const redisConfig = config.get("redisForTenantCache");
```

Comment:
@zhangxin511 I think the idea is to create a new configuration called `redisForTenantCache`, which allows for more flexibility in case we want it to be different than the Redis cache used for other purposes.

Reply:
I don't have strong opinions, but I don't prefer to maintain something won't exists yet and we are not sure if we are going to do.

This is why I think we don't need another redis:
1. I was the "cache owner" in previous companies, and I am pretty sure we have much larger room in redis to serve our data volume. 
2. Beside, redis has a concept of database, which contains 16. If we really want to separate the logic, we can use different of databases instead of new instances, which is way too expensive.  You can reference how database can be used in https://nickcraver.com/blog/2019/08/06/stack-overflow-how-we-do-app-caching/

With that being said, this new section is not a blocking change, but I don't think we really need to maintain something not exist and may not exist forever.

Reply:
I think it is a good idea to write the code in a way that could allow us to specify a different Redis instance if we want it. Behind the scenes, in the config, the `redisForTenantCache` section would just point to the same redis configuration (URL, port, password, etc) currently used in other parts of the service. I don't think adding this new config adds any overhead in terms of maintenance or anything. So, my suggestion would be to keep what you did here with `redisForTenantCache`.

One note for @kekachmar : I see you already updated config.json in routerlicious/config. You would also need to update the fluid-configmap.yaml file, which generates the config.json file in the Kubernetes context used by OSS. Later, when we do an OSS bump in FRS, you will have to make similar changes to the config.json and fluid-configmap.yaml files we have on the FRS side.

Reply:
Thank you. I updated fluid-configmap.yaml with the redisForTenantCache information